src/modules/displayfood.js

```diff
@@ -7,7 +7,7 @@ export const display = async () => {
 /* eslint-enable */
   // let markup = '';
```

Comment:
I suggest that you do not commit old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history.